src/Microsoft.AspNetCore.Blazor/Json/JsonUtil.cs

```diff
@@ -28,5 +37,30 @@ public static string Serialize(object value)
         /// <returns>An object of the specified type.</returns>
         public static T Deserialize<T>(string json)
             => SimpleJson.SimpleJson.DeserializeObject<T>(json);
+
+        /// <summary>
+        /// Deserializes the JSON string, creating an object of the specified generic type.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="json">The JSON string.</param>
+        /// <param name="propertyNaming">The property naming of the JSON string.</param>
+        /// <returns>An object of the specified type.</returns>
+        public static T Deserialize<T>(string json, PropertyNaming propertyNaming)
+            => SimpleJson.SimpleJson.DeserializeObject<T>(json, (int)propertyNaming);
+
+        /// <summary>
+        /// Property Naming scheme to be used by simple json when deserializing or serializing JSON.
+        /// </summary>
+        public enum PropertyNaming
```

Comment:
Actually could we rename this to `JsonPropertyNaming`? Otherwise it's ambiguous whether we're referring to the JS property names or the C# ones.

Reply:
Let's try to make it clearer still:

```csharp
public enum PropertyNameConversion
{
    /// <summary>
    /// Indicates that the property names should be left unchanged.
    /// </summary>
    None,

    /// <summary>
    /// Indicates that the JSON representation should use camel case.
    /// </summary>
    JsonCamelCase
}
```